src/conda_package_handling/tarball.py

```diff
@@ -2,12 +2,18 @@
 import re
 import subprocess
 import sys
+import tarfile
 from tempfile import NamedTemporaryFile
 
-import libarchive
+try:
+    import libarchive
+    libarchive_enabled = True
```

Comment:
Should we log a warnings or debug message here?

Reply:
Is it necessary?

Reply:
I would like to, but experience seems to indicate that warnings only serve to upset users.  Best to leave them with slower speed and peace and quiet, so that we are left with peace and quiet.